cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/StreamHandler.java

```diff
@@ -266,6 +274,7 @@ public void setConfig(HttpRequest request, HttpResponder responder,
                         @PathParam("stream") String stream) throws Exception {
 
     String accountId = getAuthenticatedAccountId(request);
+    Id.Stream streamId = Id.Stream.from(accountId, stream);
```

Comment:
``namespaceId``, well maybe you can do this one later since there's no concept of namespaces here.